src/Stripe.net/Services/Subscriptions/SubscriptionListOptions.cs

```diff
@@ -10,6 +10,30 @@ public class SubscriptionListOptions : ListOptionsWithCreated
         [JsonProperty("billing")]
         public Billing? Billing { get; set; }
 
+        /// <summary>
+        /// A filter on the list based on the object current_period_end field.
+        /// </summary>
+        [JsonProperty("current_period_end")]
+        public DateTime? CurrentPeriodEnd { get; set; }
+
+        /// <summary>
+        /// A filter on the list based on the object current_period_end field.
+        /// </summary>
+        [JsonProperty("current_period_end")]
+        public DateRangeOptions CurrentPeriodEndRange{ get; set; }
```

Comment:
Quite minor, but this could use another space between `Range` and the opening curly brace.